ARCHITECTURE.md

```diff
@@ -0,0 +1,94 @@
+# Standard Design and Architecture
+
+At the time of writing, almost a year of exploratory and freestyle project history has past.
+Fortunately, it is not necessary for further understanding, so I'll spare you that.
+This document, though, lays out the design, architecture and direction of Standard.
+
+If the topcis discussed herein are dear to you, please take it as an invitation to get involved.
+
+This design document can only be altered through an RFC process.
+
+## Overview
+
+Standard is a _kitchen sink_ to bootsrap and sustain an _efficient_ Software Delivery Lifecycle (SDLC) with the power of Nix and Flakes.
+_Kitchen sink_, here, is a coloquial word for _Horizontal Integration Framework_ which integrates _vertical_ tooling, that is designed around the linux philosophy of doing one thing and to it well.
+The integration target, thereby, is the _efficient_ SDLC end-to-end process for which we offer suitable and well-integrated tools and best practices.
+An _efficient_ SDLC is characterized by two things.
+Firstly, an adequate _lead time_ which is the amount of time it takes to set up an initial version of the software delivery pipeline.
+It needs to be "adequate" rather than "just fast", because it takes place in the context of a team and thus encompasses learning and onboarding activities.
+Rather than optimizing for speed, these need optimization for success, above all.k
+Secondly, an efficient SDLC is characterized by short _cycle times_ which is the amount of time it takes for a commit to be shipped to a production environment.
+Along this journey, we encounter aspects of the development, or in the broaders sense, the _contribution_ environment.
```

Comment:
More Thought needed on this. Since the goal is to defer (and out-scope) this, it's not a necessary definition, but leaves an open question for which answer is to be sought elsewhere (and we hopefully know where and can point to soon).

Reply:
You can still use the term, it just needs to be well-defined ahead of time and maybe even mentioned that it's "std specific" terminology so people don't go looking elsewhere for a definition.

Reply:
Yeah a little introductory paragraph for that term is a good idea. And if it's getting too involved.or distracting, we may also compromise back on "developer environmemt".

Reply:
I think I found a good way.